portfolio.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Georgia Adam</title>
+  <link rel="shortcut icon" href="/assets/forest.png">
+  <link rel="stylesheet" href="/styles/style.css">
+  <link rel="stylesheet" href="/styles/portfolio.css">
+
+</head>
+
+<body>
+  <nav>
+    <ul>
+      <li><a href="/index.html"> Home </a></li>
+      <li><a href="/portfolio.html"> Portfolio </a></li>
+      <li><a href="/about.html"> About </a></li>
+    </ul>
+  </nav>
+  
+  <header>
+    <h1>Georgia Adam</h1>
+    <h2>Software Engineering Student @ Ada Developers Academy</h2>
+  </header>
+
+  <section id="blurb">
+    <h3>
+      Please find a selection of projects I have worked on below.
+    </h3>
+  </section>
+
+  <main class="flex_container">
+    <div><a href="https://github.com/georgia-adam/task-list-api"><h2>Task List</h2></a><h3>Flask API</h3></div>
+    <div><a href="https://github.com/georgia-adam/book-review-site"><h2>Book Review Site</h2></a><h3>HTML/CSS</h3></div>
+    <div><a href="https://github.com/georgia-adam/swap-meet"><h2>Swap Meet</h2></a><h3>Python OOP</h3></div>
+    <div><a href="https://github.com/georgia-adam/viewing-party"><h2>Viewing Party</h2></a><h3>Python</h3></div>
```

Comment:
- We should avoid `div` unless it is purely for styling. Since each of these `divs` seems to have meaning through grouping together a block of content that represents a project, I would strongly suggest replacing the `div` tags with something like `article` or `section`.

- This is conceptually a list of projects, so I would consider wrapping them in a list element and styling as needed to achieve the desired layout. 

- There are several tags on the same line, I think it would help the structure be clearer when reading the HTML if we broke these down into multiple lines:
```html
    <div>
        <a href="https://github.com/georgia-adam/task-list-api">
            <h2>Task List</h2>
        </a>
        <h3>Flask API</h3>
    </div>
```